test/cppsuite/test_harness/workload_tracking.h

```diff
@@ -30,39 +30,82 @@
 #define WORKLOAD_TRACKING_H
 
 /*
- * Default schema for tracking table key_format : Collection name / Key value_format : Operation
- * type / Value / Timestamp
+ * Default schema for tracking table key_format : Collection name / Key / Timestamp value_format :
+ * Operation type / Value
  */
-#define DEFAULT_TRACKING_KEY_FORMAT WT_UNCHECKED_STRING(Si)
-#define DEFAULT_TRACKING_VALUE_FORMAT WT_UNCHECKED_STRING(iSi)
-#define DEFAULT_TRACKING_TABLE_SCHEMA \
-    "key_format=" DEFAULT_TRACKING_KEY_FORMAT ",value_format=" DEFAULT_TRACKING_VALUE_FORMAT
+#define DEFAULT_TRACKING_COLLECTION_KEY_FORMAT WT_UNCHECKED_STRING(Sii)
+#define DEFAULT_TRACKING_COLLECTION_VALUE_FORMAT WT_UNCHECKED_STRING(iS)
+#define DEFAULT_TRACKING_COLLECTION_TABLE_SCHEMA         \
```

Comment:
The names can be improved, I think we just want to call one SCHEMA_TRACKING and the other OPERATION_TRACKING. e.g.
```
#define SCHEMA_TRACKING_KEY_FORMAT
#define SCHEMA_TRACKING_VALUE_FORMAT

#define OPERATION_TRACKING_KEY_FORMAT
#define OPERATION_TRACKING_VALUE_FORMAT
``